AccessCheckoutDemo/AccessCheckoutDemo/CardFlowViewController.swift

```
@@ -182,17 +177,21 @@ class CardFlowViewController: UIViewController {
 
         resetCard(preserveContent: false, validationErrors: nil)
 
-        let validationConfig = try! CardValidationConfig.builder()
+        // The accessCheckoutClient is initialised here and reused within the submitCard method
+        self.accessCheckoutClient = try? AccessCheckoutClientBuilder()
+            .accessBaseUrl(Configuration.accessBaseUrl)
             .checkoutId(Configuration.checkoutId)
+            .build()
+
+        let validationConfig = try! CardValidationConfig.builder()
             .pan(panTextField)
             .expiryDate(expiryDateTextField)
             .cvc(cvcTextField)
-            .accessBaseUrl(Configuration.accessBaseUrl)
             .validationDelegate(self)
             .enablePanFormatting()
             .build()
 
-        AccessCheckoutValidationInitialiser().initialise(validationConfig)
+        accessCheckoutClient?.initialiseValidation(validationConfig)
```

The accessCheckoutClient is an important part of this logic.

Therefore the `accessCheckoutClient` should not be optional we should expect the builder to succeed.
Dhould use try! instead of try? to ensure it is non-optional and fail fast on error.

try! will crash if an error is thrown, but guarantees a non-optional value.

this also simplifies the check we have on line 41:  `self.accessCheckoutClient?`